### PR TITLE
Allow non-scan based FMRC ojbects

### PR DESCRIPTION
**Pull request by lesserwhirls**
Allow explicility listed files in NcML FMRC to be read in as FMRC objects

Allow https namespaces in NcMLCollectionReader

### CODE REVIEW

**Comment by lesserwhirls** [comment]
Will start a run on Jenkins to make sure life is good-ish.

**Comment by lesserwhirls** [comment]
Ok, no new failures on Jenkins, so I think we're good.

File: cdm/src/test/java/ucar/nc2/ncml/TestAggFmrc.java [hunk 1]
```diff
@@ -159,6 +176,95 @@ public void testAggDataValuesWithOrig() throws IOException {
...
+
+    try {
+      List<CalendarDate> scanRunTimes = fmrcScan.getRunDates();
+      List<CalendarDate> explicitRunTimes = fmrcExplicit.getRunDates();
+
+      Assert.assertEquals(scanRunTimes.size(), 3);
+      Assert.assertEquals(explicitRunTimes.size(), 3);
+
+      for (int i = 0; i < 3; i++) {
+        scanRunTimes.get(i).compareTo(explicitRunTimes.get(i));
```
Comment (cwardgar) [inline cdm/src/test/java/ucar/nc2/ncml/TestAggFmrc.java]: This is a no-op.

File: cdm/src/main/java/thredds/inventory/NcmlCollectionReader.java [hunk 2]
```diff
@@ -108,12 +90,12 @@ static public NcmlCollectionReader open(String ncmlLocation, Formatter errlog) t
...
-    if (scanElem == null) {
-      errlog.format("NcML must have aggregation scan or scanFmrc element");
-      return null;
-    }
+    Element scanElem = aggElem.getChild("scan", myNS);
+    //if (scanElem == null) scanElem = aggElem.getChild("scanFmrc", Catalog.ncmlNS);
+    //if (scanElem == null) {
+    //  errlog.format("NcML must have aggregation scan or scanFmrc element");
+    //  return null;
+   //}
```
Comment (cwardgar) [inline cdm/src/main/java/thredds/inventory/NcmlCollectionReader.java]: Can this block be removed? Commented-out code is a form of technical debt that we have a lot of already.

File: cdm/src/test/java/ucar/nc2/ncml/TestAggFmrc.java [hunk 3]
```diff
@@ -159,6 +176,95 @@ public void testAggDataValuesWithOrig() throws IOException {
...
+
+    try {
+      List<CalendarDate> scanForecastTimes = fmrcScan.getForecastDates();
+      List<CalendarDate> explicitForecastTimes = fmrcExplicit.getForecastDates();
+
+      Assert.assertEquals(scanForecastTimes.size(), 24);
+      Assert.assertEquals(explicitForecastTimes.size(), 24);
+
+      for (int i = 0; i < 3; i++) {
+        scanForecastTimes.get(i).compareTo(explicitForecastTimes.get(i));
```
Comment (cwardgar) [inline cdm/src/test/java/ucar/nc2/ncml/TestAggFmrc.java]: This too.

**Comment by lesserwhirls** [comment]
Good catch on the no-ops...uncovered a bug dealing with the correct identification of runtime, and that the `coord` attribute, which is the explicit runtime value for each explicitly listed dataset, was not being applied correctly. w00t. I'll also re-run jenkins, to be safe.

**Comment by lesserwhirls** [comment]
Ok, again, no new failures. Yay.

**Pull request merged**